tests/Aspire.Hosting.Elasticsearch.Tests/ElasticsearchFunctionalTests.cs

```diff
@@ -219,7 +252,7 @@ public async Task VerifyWaitForOnElasticsearchBlocksDependentResources()
         var resource = builder.AddElasticsearch("resource")
                               .WithHealthCheck("blocking_check");
 
-        var dependentResource = builder.AddElasticsearch("dependentresource")
+        var dependentResource = builder.AddNats("nats")
```

Comment:
```suggestion
        var dependentResource = builder.AddElasticsearch("dependentresource")
```
This doesn't seem right.

Reply:
Other tests use nginx because it's fast. I wanted to use nats because it's even smaller (and it doesn't fail on my machine due to container cache service)

example:

```
        var resource = builder.AddAzureCosmosDB("resource")
                              .RunAsEmulator()
                              .WithHealthCheck("blocking_check");

        var dependentResource = builder.AddContainer("nginx", "mcr.microsoft.com/cbl-mariner/base/nginx", "1.22")
                                       .WaitFor(resource);
```

Reply:
In this case, the previous code was using two elasticsearch containers to test the dependency checks, ES is already slow to start, requiring two of them is too much.

Reply:
Having unnecessary/surprising references is confusing and potentially a maintainence problem in the future. It would be best if all the tests were consistent.

I can help you fix the nginx problem on your machine, if you need.